Historian/Locale.js

```diff
@@ -491,7 +491,7 @@ var locale = {
 		gemGrow13: "I",
 		gemGrow14: "P",
 		gemGrow15: "I",
-		gemGrow16: "N",
+		gemGrow16: "T",
```

Comment:
Gemstone precip-INATOR is meant to be spelled that way

Reply:
Replaced N back into PRECIPINATOR.

Reply:
Again not sure if pull automatically updates this.